docs/build/iota-sdk/1.0/docs/how-tos/accounts-and-addresses/consolidate-outputs.mdx

```diff
@@ -69,40 +69,29 @@ https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/how_tos/account
 <div className={'hide-code-block-extras'}>
 
 ```typescript reference
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts#L51
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts#L48
 ```
 
 </div>
 
-3. Prepare a [`Transaction`](../../references/nodejs/classes/Transaction.md) to consolidate the Outputs by calling
-   the [`Account.prepareConsolidateOutputs()`](../../references/nodejs/classes/Account.md#prepareconsolidateoutputs) function.
-
-<div className={'hide-code-block-extras'}>
-
-```typescript reference
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts#L70-L72
-```
-
-</div>
-
-4. Send the `Transaction` by calling the [`PreparedTransaction.send()`](../../references/nodejs/classes/PreparedTransaction.md#send)
-   function.
+3. Send a [`Transaction`](../../references/nodejs/classes/Transaction.md) to consolidate the Outputs by calling
+   the [`Account.consolidateOutputs()`](../../references/nodejs/classes/Account.md#consolidateoutputs) <!-- TODO Update ref link --> function.
```

Comment:
```suggestion
   the [`Account.consolidateOutputs()`](../../references/nodejs/classes/Account.md#consolidateoutputs) function.
```

Reply:
We should leave them until I fix the refs